src/app/home/about/about.component.scss

```diff
@@ -0,0 +1,21 @@
+#layer {
+  background-color: rgba(100, 181, 246, 0.45);
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  z-index: 1;
+}
+.title-about-color {
+  color: #64B5F6;
+  font-size: 20pt;
+  font-weight: bold;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+.detail-about {
+  font-size: 15pt;
+  font-weight: normal;
```

Comment:
Avoid this, try just using p element or some h4, h5 or some of them if you prefer.